### ISSUE TITLE
Fractured Space (310380)

### ISSUE THREAD

**Issue by h8672** [ValveSoftware/Proton#1584]
# Compatibility Report
- Name of the game with compatibility issues: Fractured Space
- Steam AppID of the game: 310380

## System Information
- GPU: GTX 1060 6GB
- Driver/LLVM version: nvidia 396.54
- Kernel version: 4.15.0-34-generic
- Link to full system information report: [Gist](https://gist.github.com/h8672/b99bb02dbff7ac920f756944433f09a4)
- Proton version: 4.7-6

## I confirm:
- [ X ] that I haven't found an existing compatibility report for this game.
- [ X ] that I have checked whether there are updates for my system available.

[steam-310380.log](https://github.com/ValveSoftware/Proton/files/2411276/steam-310380.log)

Log file from starting launcher, game gives 21.7 MB log, and trying to create lobby gives 45.7 MB log if I quit fast... waiting grows log more. Solo AI vs AI gives 111,3 MB log after it returns to hangar on its own.

They don't fit as attachment has 10 MB limit. Here's a zip. [steam-310380Game.zip](https://github.com/ValveSoftware/Proton/files/2411409/steam-310380Game.zip) Contains 4 logs, total size of 282.6 MB and as a zip it's 7.7 MB.

## Symptoms 
1. Lag in the game. Fix PROTON_NO_ESYNC fixes it, but models load slowly at start still (maybe cause of log command in launch script).
2. Cannot create a lobby. Fix get invite from someone to lobby.
3. if can't get past launcher, add launch script "-autolaunch" to steam options to skip launcher.

## Reproduction
Install game. launch it, try to go battle.
Restart game if infinite loop, closing game from desktop gives a verification popup in game.
Get an invite from a friend and enter battle with him.

**Comment by Flaburgan** [comment]
Hi there. How did you manage to launch the game? `-autolaunch` is not doing anything to me, the launcher is white. RX580 with kernel 4.18, Mesa 18.2.3, proton 3.16-3 beta.

**Comment by h8672** [comment]
I'm out for a week or so, so I cannot test the new version yet.
I was using Nvidia GTX 1060, so it might be little different, but I added the launchscript through steam.
Steam Library -> Fractured Space, properties -> Set Launch Options -> -autolaunch

**Comment by Flaburgan** [comment]
Yep, that's what did. I'll wait for kernel 19.1 to be released and will try it but I doubt it will make a difference. Can anyone tell me how I can provide feedback / logs / anything to help debugging?

**Comment by h8672** [comment]
Add `PROTON_LOG=1 %command%` to the game's launch options and drop the generated `$HOME/steam-$APPID.log`. Id for Fractured Space is 310380, which can be seen through game store page in address https://store.steampowered.com/app/310380.

**Comment by h8672** [comment]
Just tried using 3.16-6 Beta with nvidia 415.25 drivers and I could play it without additional changes. Coop lobby could be opened and used to start a game but solo didn't start and just returned back to shipyard.
